tests/test-definition.yaml

```diff
@@ -18,24 +16,24 @@ dimensions:
       - 1.15.3-stackable0.0.0-dev
       - 1.16.3-stackable0.0.0-dev
       - 1.18.0-stackable0.0.0-dev
+      - 1.20.0-stackable0.0.0-dev
+      - 1.21.0-stackable0.0.0-dev
   - name: nifi_old
     values:
-      - 1.15.3-stackable0.0.0-dev
-      - 1.16.3-stackable0.0.0-dev
+      - 1.18.0-stackable0.0.0-dev
   - name: nifi_new
     values:
-      - 1.18.0-stackable0.0.0-dev
+      - 1.21.0-stackable0.0.0-dev
   - name: nifi-latest
     values:
-      - 1.18.0-stackable0.0.0-dev
+      - 1.21.0-stackable0.0.0-dev
   - name: ldap-use-tls
     values:
       - "false"
       - "true"
   - name: openshift
     values:
       - "false"
-  # Used for both, zookeeper and nifi
```

Comment:
Why did you remove this comment? For me personally it helps

Reply:
ah i actually wanted to remove the number of tests and remove it from zookeeper but it would not have done that.... will put it back.